### DISCUSSION TITLE
Clicking the pencil button on a preset docs page takes you to source file

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#25229] [Suggest an Idea]
### Tell us more.

Right now our pencil button is broken for some docs pages:

- #19309

But the pencil button never worked for _presets_. It would be awesome if you could just click the pencil button on any preset page, and go to the correct source file. This makes it easier to see something wrong and fix it.

I'm opening this as a discussion first, to get feedback from the maintainers. 😄

**Comment by rarkins** [comment]
Is the pencil button per-page or per-preset?

**Reply by HonkingGoose** [reply]
The pencil button is _per-page_. So selecting the button takes you to the source file on GitHub, then search/scroll to the correct place yourself.

**Reply by rarkins** [reply]
I guess we need to add the link manually for each then, to one of the files in https://github.com/renovatebot/renovate/tree/main/lib/config/presets/internal

**Reply by HonkingGoose** [reply]
Sounds good to me! Do you want me to create an issue then?

**Reply by rarkins** [reply]
#25236
